### DISCUSSION TITLE
ROS Foxy 사용 시, 새로운 워크스페이스를 자동 바라보기~/ros_menu/config.yaml 오류

### DISCUSSION THREAD

**Discussion by khehaeun** [Road-Balance/gcamp_ros2_basic#22] [Q&A] [answered]
![23페이지오류](https://user-images.githubusercontent.com/108414567/178228788-4e8d0acd-4fd3-4e71-861c-cfc45005a0dd.png)

mkdir -p ~/ros2_ws/src
cd ~/ros2_ws/src
cd ~/ros2_ws
colcon build --symlink-install
입력 후 
 gedit ~/ros_menu/config.yaml
입력하였습니다.

gedit 오류가 뜨면서 gedit을 설치하라는 문구가 떠 설치 후 재입력을 하였으나 
 위 사진과 같은 오류가 나와 도움을 요청드리고 싶습니다.

**Comment by kimsooyoung** [comment, marked as answer]
강의 중 이야기되었지만 gedit 설정은 이후 실행에 문제 없습니다.
